src/main/resources/fuse_regression/test_1.yaml

```diff
@@ -0,0 +1,74 @@
+---
+corpus: beir-v1.0.0-robust04.flat
+corpus_path: collections/beir-v1.0.0/corpus/robust04/
+
+metrics:
+  - metric: nDCG@10
+    command: bin/trec_eval
+    params: -c -m ndcg_cut.10
+    separator: "\t"
+    parse_index: 2
+    metric_precision: 4
+    can_combine: false
+  - metric: R@100
+    command: bin/trec_eval
+    params: -c -m recall.100
+    separator: "\t"
+    parse_index: 2
+    metric_precision: 4
+    can_combine: false
+  - metric: R@1000
+    command: bin/trec_eval
+    params: -c -m recall.1000
+    separator: "\t"
+    parse_index: 2
+    metric_precision: 4
+    can_combine: false
+
+topic_reader: TsvString
+topics:
+  - name: "BEIR (v1.0.0): Robust04"
+    id: test
+    path: topics.beir-v1.0.0-robust04.test.tsv.gz
+    qrel: qrels.beir-v1.0.0-robust04.test.txt
+
+# Fusion Regression Test Configuration
+runs:
```

Comment:
Why don't we assume that the runs are generated by a regression in `https://github.com/castorini/anserini/tree/master/src/main/resources/regression`? E.g., `beir-v1.0.0-robust04.flat.yaml`

So we keep the yaml file and the run name (i.e., just take the run file name it generates).

Reply:
I established a test_2.yaml, do you mind taking a look to see if that is something you'd like? 
## Perform two regression runs for test2
One could generate the runs necessary for test 2 following 
- https://github.com/castorini/anserini/blob/master/docs/regressions/regressions-beir-v1.0.0-robust04.bge-base-en-v1.5.flat-int8.cached.md
- https://github.com/castorini/anserini/blob/master/docs/regressions/regressions-beir-v1.0.0-robust04.bge-base-en-v1.5.flat.cached.md

Or do you want the fusion yaml file to contain information about the yaml files (the ones that produce runs for the fusion test) it depends on and perform those runs automatically when the fusion test is performed?

Reply:
Its name is now changed into `src/main/resources/fuse_regression/fusion-regression-bge-flat-int8-robust04-2.yaml`